chain/chain/src/resharding/trie_state_resharder.rs

```diff
@@ -160,6 +163,7 @@ impl TrieStateResharder {
             child.next_key.clone(),
             &mut store_update.trie_store_update(),
             expect_memtries,
+            &status.resharding_block_hash,
```

Comment:
Does it mean that the TrieChanges of resharding_block_hash accumulate the entire state of both child shards? If so, I wonder how long it would take to GC or copy the resharding block to cold storage, and whether that would impact normal node operation. Something to be checked in forknet.

Reply:
oh my, you're right, I put it in the wrong place. Good catch!